File: car-prices-backend/car-prices-backend/Models/Request/VehicleRequest.cs
namespace car_prices_backend.Models.Request
{
    public enum vehicleTypeEnum
    {
        Common,
        Luxury
    }
    public class VehicleRequest
    {
        public string VehicleBasePrice { get; set; }
        public vehicleTypeEnum VehicleType { get; set; }
    }
}

File: front-angular/src/app/models/vehicleRequest.ts
export enum vehicleTypeEnum {
    Common,
    Luxury
}

export class VehicleRequest {
    vehicleBasePrice!: number;
    vehicleType!: vehicleTypeEnum;

    public constructor(init?: Partial<VehicleRequest>) 
    {
        Object.assign(this, init);
    }
}